### ISSUE TITLE
What is the nature of dark matter?

### ISSUE THREAD

**Issue by cwjames1983** [ethrane/transients#8]
[I'm raising this because it's clearly a huge issue, and I'm not 100% sure if it's going to be covered. Is anybody on "Galaxies" here?]

Dark matter consists of 25% of the Universe, gets studied a lot by multimessenger experiments (gamma rays at GeV and TeV and  neutrinos). It's a topic for significant infrastructure investment.

**Comment by SimonStevenson** [comment]
Dark matter is certainly very important, but I'm not sure this group is the right place for it.

**Comment by cwjames1983** [comment]
> Dark matter is certainly very important, but I'm not sure this group is the right place for it.

It's the multimessenger group, and it's multimessenger physics. I do agree that there is much more "transients" in Australia than there is "multimessenger", hence I fully understand why its priority would be lower though.

**Comment by anaismoller** [comment]
Both dark matter and dark energy are linked to transients and multi-messenger as probes. What would be the difference between our question and the cosmology group?

**Comment by ethrane** [comment]
I agree with @anaismoller. I think dark matter and dark energy are (hugely) important, and I think transients can play a role, but Group 1.1 (Galaxies & Cosmology) is sure to focus on dark matter and dark energy.  My advice is for us to work with them to make sure that transients are part of their key question. However, I prefer to use our report to highlight science priorities that are more unambiguously in the transient wheelhouse.

**Comment by cwjames1983** [comment]
@ethrane I think we should then ensure we *do* contact that group before drawing any conclusions. After all, why would they not say that multimessenger dark matter searches are part of the multimessenger group?

**Comment by cwjames1983** [comment]
@ethrane message sent to Scott Croom, head of the cosmology group (which I'm on, with an FRB hat on). Let's see the response.
